core/contract/src/main/java/org/eclipse/dataspaceconnector/contract/listener/ContractNegotiationEventListener.java

```diff
@@ -0,0 +1,108 @@
+/*
+ *  Copyright (c) 2022 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Bayerische Motoren Werke Aktiengesellschaft (BMW AG) - initial API and implementation
+ *
+ */
+
+package org.eclipse.dataspaceconnector.contract.listener;
+
+import org.eclipse.dataspaceconnector.spi.contract.negotiation.observe.ContractNegotiationListener;
+import org.eclipse.dataspaceconnector.spi.event.EventRouter;
+import org.eclipse.dataspaceconnector.spi.event.contractnegotiation.ContractNegotiationApproved;
+import org.eclipse.dataspaceconnector.spi.event.contractnegotiation.ContractNegotiationConfirmed;
+import org.eclipse.dataspaceconnector.spi.event.contractnegotiation.ContractNegotiationDeclined;
+import org.eclipse.dataspaceconnector.spi.event.contractnegotiation.ContractNegotiationFailed;
+import org.eclipse.dataspaceconnector.spi.event.contractnegotiation.ContractNegotiationInitialized;
+import org.eclipse.dataspaceconnector.spi.event.contractnegotiation.ContractNegotiationOffered;
+import org.eclipse.dataspaceconnector.spi.event.contractnegotiation.ContractNegotiationRequested;
+import org.eclipse.dataspaceconnector.spi.types.domain.contract.negotiation.ContractNegotiation;
+
+import java.time.Clock;
+
+public class ContractNegotiationEventListener implements ContractNegotiationListener {
+    private final EventRouter eventRouter;
+    private final Clock clock;
+
+    public ContractNegotiationEventListener(EventRouter eventRouter, Clock clock) {
+        this.eventRouter = eventRouter;
+        this.clock = clock;
+    }
+
+    @Override
+    public void initialized(ContractNegotiation negotiation) {
```

Comment:
nit: shouldn't this be called `initiated`? At least that's the term we use in most other places.

Reply:
you're right, I'm not fully aware of the difference between the two terms, anyway corrected :+1: